python/cudf/cudf/core/dataframe.py

```diff
@@ -7709,6 +7709,52 @@ def equals(self, other):
                 return False
         return super().equals(other)
 
+    def explode(self, column, ignore_index=False):
+        """
+        Transform each element of a list-like to a row, replicating index
+        values.
+
+        Parameters
+        ----------
+        column : str or tuple
+            Column to explode.
+        ignore_index : bool, default False
+            If True, the resulting index will be labeled 0, 1, …, n - 1.
+
+        Returns
+        -------
+        DataFrame
+
+        Examples
+        -------
```

Comment:
```suggestion
        Examples
        --------
```

Reply:
Where's the rule for title separator? I can see inconsistent usage in other places of the code.

Reply:
If I remember correctly, Ideally it has to match with the number of characters of the title, else sphinx throws some warnings